packages/purpose-process/src/services/purposeService.ts

```diff
@@ -837,6 +840,41 @@ export function purposeServiceBuilder(
         isRiskAnalysisValid,
       };
     },
+    async retrieveRiskAnalysisConfigurationByVersion({
+      eserviceId,
+      riskAnalysisVersion,
+      organizationId,
+    }: {
+      eserviceId: EServiceId;
+      riskAnalysisVersion: string;
+      organizationId: TenantId;
+    }): Promise<RiskAnalysisFormRules> {
+      logger.info(
+        `Retrieve version ${riskAnalysisVersion} of risk analysis configuration`
+      );
+
+      const eservice = await retrieveEService(eserviceId, readModelService);
+      const tenant = await getInvolvedTenantByEServiceMode(
+        eservice,
+        organizationId,
+        readModelService
+      );
+
+      assertTenantKindExists(tenant);
+
+      const riskAnalysisFormConfig = riskAnalysisFormRules[tenant.kind].find(
+        (config) => config.version === riskAnalysisVersion
+      );
+
+      if (!riskAnalysisFormConfig) {
+        throw RiskAnalysisConfigVersionNotFound(
+          riskAnalysisVersion,
+          tenant.kind
+        );
+      }
```

Comment:
> Leaving my 2 cents since I implemented the current risk analysis validation and saw this PR pass by: what do you think about moving this logic into the `commons` risk analysis package?
> 
> We already have a `getLatestVersionFormRules` method in `riskAnalysisValidation.ts`, we could also add a `getFormRulesByVersion` there (and test it in commons-test), so that we avoid exporting all the rules files outside the module, and we keep all the risk analysis related logic into its dedicated commons module

It makes totally sense, also because this logic could probably be reused. I'll change it soon!

Reply:
@ecamellini I implemented as you suggested with two differences:

1. I kept the thowing in the endpoint logic, making `getFormRulesByVersion` return `undefined` like `getLatestVersionFormRules` does;
2. I had to maintain the export of the rules files in order to test it in `common-test`.

Let me know what you think!